docs/installation.rst

```diff
@@ -8,13 +8,13 @@ Installation
 Stable release
 --------------
 
-To install google_cl, run this command in your terminal:
+To install googlecl, run this command in your terminal:
 
 .. code-block:: console
 
-    $ pip install google_cl
+    $ pip install googlecl
```

Comment:
Thanks for contributing. I don't have access to the `googlecl` on pypi but the `google_cl`. So if you can update this, I can merge your pull request.